includes/admin/tools/export/give-export-donations-exporter.php

```diff
@@ -100,6 +100,13 @@ public function set_properties( $request ) {
 		$this->start      = isset( $request['start'] ) ? sanitize_text_field( $request['start'] ) : '';
 		$this->end        = isset( $request['end'] ) ? sanitize_text_field( $request['end'] ) : '';
 		$this->status     = isset( $request['status'] ) ? sanitize_text_field( $request['status'] ) : 'complete';
+
+		/**
+		 * Hook to use after setting properties.
+		 *
+		 * @since 2.1.3
+		 */
+		do_action( 'give_donations_export_form_data', $this->data );
```

Comment:
@Sidsector9  rename `give_donations_export_form_data` to `give_export_donations_form_data`